client/src/pages/EditProfile.js

```diff
@@ -144,13 +155,25 @@ export default function EditProfile() {
                                 className={classes.profileTopLeft}
                             >
                                 <Grid item xs={12}>
-                                    <Box
-                                        boxShadow={2}
-                                        component={Avatar}
-                                        src={sampleUser}
-                                        alt="user"
-                                        className={classes.userImage}
-                                    />
+                                    {!user.profile.profilePicURL ? (
+                                        <Box
+                                            boxShadow={2}
+                                            component={Avatar}
+                                            src={user.profile.profilePicURL}
```

Comment:
Is this supposed to be a different source?
Rather than conditionally having the whole box, you can just conditionally assign the src:
`src={user.profile.profilePicURL ? user.profile.profilePicURL : defaultImg }`

Reply:
I was trying to get the material ui avatar to show a person icon if there was no image.  According the the docks if there is no image it defaults to the first letter of the alt and if there is no alt it defaults to the person icon.  So what I want is  alt={user.profile.profilePicURL ? "profile image" : ""}